sanity/schemas/globalContent.ts

```diff
@@ -5,6 +5,15 @@ export default defineType({
   name: "global-content",
   type: "document",
   fields: [
+    defineField({
+      title: "Integrations",
+      name: "integrations",
+      type: "string",
+      initialValue: "manual",
+      options: {
+        list: ["manual", "openAPI", "uniform"],
```

Comment:
can we remove uniform until its a requirement

Reply:
Removed ✨